knx.js

```diff
@@ -214,6 +214,16 @@ module.exports = function (RED) {
                 case '8':    //16-bit signed value                2 Byte                  DPT 8         DPT 8    -32768...32767
                 case '12':   //32-bit unsigned value              4 Byte                  EIS 11        DPT 12    0...4294967295
                 case '13':   //32-bit signed value                4 Byte                  DPT 13        DPT 13    -2147483648...2147483647
+                case '16':   //String                            14 Byte                  DPT 16        DPT 16    ASCII or ISO 8859-1/Latin-1
+                    buf = Buffer.alloc(14, 0);
```

Comment:
Do not define "magic number", please.
Better create a constant:
```suggestion
                    const stringMaxLength = 14;
                    buf = Buffer.alloc(stringMaxLength, 0);
```

By the way: why exactly 14?

Reply:
No answer from author, @waldbaer :(

Reply:
Based on KNX DPT specification chapter 3.17 Datapoint Types "String":
> The length is fixed to 14 octets